models/sessions/scratch/databricks/snowplow_web_sessions_this_run.sql

```diff
@@ -287,6 +344,11 @@ select
         {%- for conv_def in var('snowplow__conversion_events') %}
     {{ snowplow_web.get_conversion_columns(conv_def, names_only = true)}}
         {%- endfor %}
+    {% if var('snowplow__total_all_conversions', false) %}
+    ,{%- for conv_def in var('snowplow__conversion_events') %}{{'cv_' ~ conv_def['name'] ~ '_volume'}}{%- if not loop.last %} + {% endif -%}{%- endfor %} as cv__all_volume
+    {# Use 0 in case of no conversions having a value field #}
+    ,0 {%- for conv_def in var('snowplow__conversion_events') %}{%- if conv_def.get('value') %} + {{'cv_' ~ conv_def['name'] ~ '_total'}}{% endif -%}{%- endfor %} as cv__all_total
```

Comment:
Cast to `int`?

Reply:
The total field could be a float or decimal, but we are relying on them being the same (or warehouses supporting adding them), we could cast the total column in the conversion definition macro, but I think we should be fine. We do call out in the docs they should make sure each column is the same unit